### DISCUSSION TITLE
v2.13.0

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#2773] [General]
> **Important:** more information about this release can be found on [our blog](https://filamentphp.com/blog/v2130-admin-resources). It will be useful to read it if you use the Admin Panel, or build CRUD tables using the Table Builder.

## What's Changed
* [ar] Translate phrases by @mohamedsabil83 in https://github.com/laravel-filament/filament/pull/2735
* chore: Deprecations by @danharrin in https://github.com/laravel-filament/filament/pull/2754
* feature: table and page action groups by @ryangjchandler in https://github.com/laravel-filament/filament/pull/2706
* button component isDisabled getter by @wychoong in https://github.com/laravel-filament/filament/pull/2740
* feature: Action modal content by @danharrin in https://github.com/laravel-filament/filament/pull/2762
* Update Swedish translations by @adevade in https://github.com/laravel-filament/filament/pull/2761
* Update wizard arabic translations by @mohammedokasha in https://github.com/laravel-filament/filament/pull/2767

## New Contributors
* @adevade made their first contribution in https://github.com/laravel-filament/filament/pull/2761
* @mohammedokasha made their first contribution in https://github.com/laravel-filament/filament/pull/2767

**Full Changelog**: https://github.com/laravel-filament/filament/compare/v2.12.31...v2.13.0

<hr /><em>This discussion was created from the release <a href='https://github.com/laravel-filament/filament/releases/tag/v2.13.0'>v2.13.0</a>.</em>

**Comment by intrepidws** [comment]
This references a [blog post](https://filamentphp.com/blog/v2130-admin-resources) that no longer seems to exist - is that content available anywhere?

**Reply by danharrin** [reply]
https://v2.filamentphp.com/blog/v2130-admin-resources
